resources/state_of_the_dapps.json

```diff
@@ -0,0 +1,43 @@
+{
+    "auction-house":
+    {
+        "name":
+        {
+            "en": "Auction House"
+        },
+        "photo-path": "contacts://auction-house",
+        "state-of-the-dapp?": true,
```

Comment:
Doesn't the fact that this contact placed inside `state_of_the_dapps.json` file automatically mean that it is `state-of-the-dapp?`?